app/components/TumourSummaryEdit/index.tsx

```diff
@@ -323,52 +378,95 @@ const TumourSummaryEdit = ({
     return null;
   }, [handleClicked, handleDelete, handleKeyDown, newMicrobialData]);
 
+  const tCellCd8DataSection = useMemo(() => {
+    return (
+      <>
+        <TextField
+          className="tumour-dialog__text-field"
+          label="CD8+ T Cell Score"
+          value={newTCellCd8Data?.score ?? 0}
```

Comment:
when the value is null, can we display no value here instead of 0? (here and in similar cases in tumour summary edit)

Reply:
react yells at you, this is a running problem, if we do want it to be null (no value) then the console of the browser will be spammed

Reply:
what's happening in the display when you delete the 0?

Reply:
empty string 
![image](https://github.com/bcgsc/pori_ipr_client/assets/25511396/013f1d95-4b66-4b55-9769-84cabda133d7)